scripts/release-candidate.sh

```diff
@@ -0,0 +1,56 @@
+#! /usr/bin/env bash
+
+set -euo pipefail
+
+# date from git tag
+tag_date() {
+  echo "${1##v}"
+}
+# cabal version from git tag
+tag_cabal_ver() {
+  tag_date "$1" | sed -e s/-0/-/g -e s/-/./g
+}
+
+BASE_COMMIT=$(curl -X GET -H "Authorization: Bearer ${BUILDKITE_API_TOKEN}" \
+    "https://api.buildkite.com/v2/organizations/cardano-foundation/pipelines/cardano-wallet/builds" \
+    | jq -r \
+        '[.[] | select(.state == "passed" and .branch == "master") | .commit][0]')
+
+echo "BASE_COMMIT=$BASE_COMMIT"
```

Comment:
Just a suggestion: if you add `x` to the `set -euox pipefail` then you don't have to echo assigned vars

Reply:
I happen to not like `x` in production. But I use it during development.